### ISSUE TITLE
Eased UI components are not styled properly since 0.11

### ISSUE THREAD

**Issue by rparrett** [vleue/bevy_easings#27]
I feel like this issue must be with Bevy, but I have been unable to reproduce without `bevy_easings` so far, so maybe not.

0.11: children are arranged in a row

<img width="1392" alt="image" src="https://github.com/vleue/bevy_easings/assets/200550/79f89888-b111-42ec-8a85-c738d51a67ab">

0.10: children are arranged in a column as intended

<img width="1392" alt="image" src="https://github.com/vleue/bevy_easings/assets/200550/3034eedd-db17-4e03-bd45-342215c42000">

```rust
use bevy::prelude::*;

use bevy_easings::*;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    App::default()
        .add_plugins(DefaultPlugins)
        .add_plugin(bevy_easings::EasingsPlugin)
        // 0.10
        //.add_startup_system(setup)
        // 0.11
        .add_systems(Startup, setup)
        .run();
    Ok(())
}

fn setup(mut commands: Commands) {
    commands.spawn(Camera2dBundle::default());

    let from_style = Style {
        flex_direction: FlexDirection::Column,
        margin: UiRect {
            top: Val::Px(0.),
            ..Default::default()
        },
        // 0.10
        //size: Size::all(Val::Px(100.)),
        // 0.11
        width: Val::Px(100.),
        height: Val::Px(100.),
        ..Default::default()
    };
    let mut to_style = from_style.clone();
    to_style.margin = UiRect {
        top: Val::Px(500.),
        ..Default::default()
    };

    commands
        .spawn((
            NodeBundle {
                background_color: BackgroundColor(Color::RED),
                style: from_style.clone(),
                ..Default::default()
            },
            from_style.ease_to(
                to_style,
                bevy_easings::EaseFunction::SineInOut,
                bevy_easings::EasingType::PingPong {
                    duration: std::time::Duration::from_secs(10),
                    pause: Some(std::time::Duration::from_millis(500)),
                },
            ),
        ))
        .with_children(|parent| {
            for _ in 0..2 {
                parent.spawn(NodeBundle {
                    background_color: BackgroundColor(Color::WHITE),
                    style: Style {
                        // 0.10
                        //size: Size::all(Val::Px(20.)),
                        // 0.11
                        width: Val::Px(20.),
                        height: Val::Px(20.),
                        ..default()
                    },
                    ..Default::default()
                });
            }
        });
}
```

**Comment by mockersf** [comment]
Oh that's my fault 🤦 

I was doing experimentations with `Style` while doing the upgrade and forgot to revert them...

**Comment by mockersf** [comment]
fixed in the 0.11.1
